File: In_C_Programming/q5.c
#include <stdio.h>

int get_element(int arr[], int ln)
{

    printf("Enter The Elements: ");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", arr[i]);
    }

    return 0;
}

int get_EvenOdd(int arr[], int ln)
{

    int even = 0, odd = 0;

    for (int i = 0; i < ln; i++)
    {
        if (arr[i] % 2 == 0)
        {
            even++;
        }
        else
        {
            odd++;
        }
    }

    printf("It have %d ", even, "Numbers");
    printf("It have %d ", odd, "Numbers");

    return 0;
}

int main()
{

    int size;

    printf("Enter The Array Size: ");
    scanf("%d", &size);

    int arr[size];

    get_element(arr, size);

    get_EvenOdd(arr, size);

    return 0;
}

File: In_Java_Programming/q5.java
import java.util.Scanner;

public class q5 {

    public static int countEvenandOdd(int arr[], int size) {

        int even = 0, odd = 0;

        for (int i = 0; i < size; i++) {

            if (arr[i] % 2 == 0) {
                even++;
            } else {
                odd++;
            }
        }

        System.out.println("The Count of Even Numbers is: " + even);
        System.out.println("The Count of Odd Numbers is: " + odd);

        return 0;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter The Array Length: ");
        int ln = sc.nextInt();

        int x[] = new int[ln];

        countEvenandOdd(x, ln);

        sc.close();
    }
}
